src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -225,6 +225,7 @@ private static function validateOptions(array $options): void
     {
         $availableOptions = array_keys(self::DEFAULT_OPTIONS);
         $availableOptions[] = 'serializer';
+        $availableOptions[] = 'auth';
```

Comment:
Why is this needed?

Reply:
Because these options did not have a default and if they aren't added to the `$avaibleOptions` array, you get a deprecation notice (exception in 6.0). 
It turns out that we can easily add ones, I did :)

Reply:
You are correct. Thank you.